tox.ini

```diff
@@ -1,8 +1,9 @@
 [tox]
 envlist =
-    python{3.7,3.8,3.9,3.10}-django{3.2}-wagtail{2.15,2.16,3.0,4.0}
-    python{3.8,3.9,3.10}-django{4.0}-wagtail{2.16,3.0,4.0}
-    python{3.8,3.9,3.10}-django{4.1}-wagtail{4.0}
+    python{3.7,3.8,3.9,3.10}-django{3.2}-wagtail{2.15,2.16,3.0,4.0,4.1,4.2}
+    python{3.8,3.9,3.10}-django{4.0}-wagtail{2.16,3.0,4.0,4.1,4.2}
+    python{3.8,3.9,3.10}-django{4.1}-wagtail{4.0,4.1,4.2}
+    python{3.11}-django{4.1}-wagtail{4.1,4.2}
```

Comment:
I've not added Django 4.2 here yet but potentially it could be installed with it. I tested it locally and it works OK